game/hiscore.cc

```diff
@@ -43,6 +41,12 @@ void Hiscore::addHiscore(HiscoreItem&& item) {
 	m_hiscore.insert(std::move(item));
 }
 
+void Hiscore::loadHiscore(HiscoreItem&& item) {
```

Comment:
`load` seams not the right term for this function. Why not `add` or if you want to differentiate from the other add-functions `store` or `put`?